bundle/tests/relative_path_with_includes_test.go

```diff
@@ -17,8 +18,12 @@ func TestRelativePathsWithIncludes(t *testing.T) {
 	diags := bundle.Apply(context.Background(), b, m)
 	assert.NoError(t, diags.Error())
 
-	assert.Equal(t, "artifact_a", b.Config.Artifacts["test_a"].Path)
-	assert.Equal(t, filepath.Join("subfolder", "artifact_b"), b.Config.Artifacts["test_b"].Path)
+	relA, err := filepath.Rel(b.SyncRootPath, b.Config.Artifacts["test_a"].Path)
+	require.NoError(t, err)
+	assert.Equal(t, "artifact_a", relA)
+	relB, err := filepath.Rel(b.SyncRootPath, b.Config.Artifacts["test_b"].Path)
+	require.NoError(t, err)
+	assert.Equal(t, filepath.Join("subfolder", "artifact_b"), relB)
```

Comment:
Artifact paths are now absolute.

Reply:
>> assert.Equal(t, filepath.Join("subfolder", "artifact_b"), relB)

> Artifact paths are now absolute.

What are you referring to? the variables in the test are relative.

Reply:
Simplified in latest.